system/prometheus-kubernetes/values.yaml

```
@@ -15,6 +15,12 @@ prometheus-server:
     enabled: true
     size: 100Gi
 
+  alerts:
+    multipleTargetScrapes:
+      exceptions:
+        - kubernetes-cadvisors
+        - kubernetes-kublet
```

Let's bring the comment
```suggestion
        # we exclude cadvisor metrics because it has the same instance as the kubelet but a different path
        # e.g. 10.246.204.80:10250/metrics vs. 10.246.204.80:10250/metrics/cadvisor
        - kubernetes-cadvisors
        - kubernetes-kublet
```